codegen.mts

```diff
@@ -57,6 +56,11 @@ const config: CodegenConfig = {
             SupportTicketPriority: '../shared/entities#SupportTicketPriority',
             SupportTicketStatus: '../shared/entities#SupportTicketStatus',
           },
+          resolversNonOptionalTypename: {
+            interfaceImplementingType: true,
+            unionMember: true,
+            excludeTypes: ['TokenInfoPayload'],
```

Comment:
This config is required because we are using [__isTypeOf in TokenInfo](https://github.com/kamilkisiela/graphql-hive/pull/4937/files#diff-90d57ac659bf1029c0543323045fe5202f300c6cdfffd232ea43f6a43d4cfecfR7-R9). Server Preset's default is using `__typename` but since all 3 options work and this is a migration, I want to keep logic the same